AGENT_CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Datadog Agent version [7.30.1](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst#7301)
+
+* There were no integration updates for this version of the Agent.
```

Comment:
There's no integration change between 7.30.0 and 7.30.1, the only change is a downgrade of `requests` and `snowflake-connector-python`: https://github.com/DataDog/integrations-core/pull/9912/files